TenureInformationApi.Tests/V1/Gateways/DynamoDbGatewayTests.cs

```diff
@@ -607,6 +655,50 @@ public async Task EditTenureDetailsWhenEndDateIsInRequestButStartDateInDatabaseI
             await act.Should().ThrowAsync<EditTenureInformationValidationException>();
         }
 
+        [Fact]
+        public async Task EditTenureDetailsWhenEndDateIsInRequestAndStartDateInDatabaseIsEqualExceptionIsNotThrown()
+        {
+            _fixture.Customizations.Add(new UtcDateTimeHelper());
+            // end date passed, start date is equal - do not throw error
+            var tenureStartDate = _fixture.Create<DateTime>();
```

Comment:
Seems a bit overkill...

1. Could just be:
```cs
var tenureStartDate = DateTime.UtcNow;
```
if you want the 'now' date.

2. Or could just use the already installed `Bogus` package:
```cs
// at the top of the class:
using Bogus;
Faker _faker = new Faker();

var tenureStartDate = _faker.Date.Past();
```
or anything from this list:
![image](https://github.com/LBHackney-IT/tenure-api/assets/43747286/d1e64c5f-d146-4e03-bed4-267e37cbf69a)

Reply:
yeah, for some reason I wanted to preserve the fixture usage
going for datetime.utcnow makes more sense and creates less noise
addressed in [6854bea](https://github.com/LBHackney-IT/tenure-api/pull/152/commits/6854bea3d3179bc648c6618ca8acdabe965cb54d)

Reply:
Going from `_fixture.Create<DateTime>()` to `DateTime.UtcNow` is of course, not equivalent.

But so long as it works fine, it should be okay.

If you do want to keep the randomness, however, using `Bogus` `Faker` is probably a way to go.